ui/pages/onboarding-flow/add-network-modal/index.js

```diff
@@ -14,7 +15,7 @@ import {
 
 import NetworksForm from '../../settings/networks-tab/networks-form/networks-form';
 
-export default function AddNetworkModal() {
+export default function AddNetworkModal({ isNewNetworkFlow = false }) {
```

Comment:
The `defaultProps` at the bottom sets this value to `true`.  Which should it be?